src/GraphQL.Harness/GraphQL.Harness.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>net6;net5;netcoreapp3.1</TargetFrameworks>
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <IsPackable>false</IsPackable>
```

Comment:
IsPackable is false by default for all Web projects.

Reply:
I can remove it here, and I can probably remove IsPackable = true in the Directory.Build.props also

Reply:
Well, it didn't work.  If I remove IsPackable = true in Directory.Build.props, then Newtonsoft doesn't think IsPackable is true.  If I remove IsPackable = false from GraphQL.Harness then the setting within Directory.Build.props of IsPackable = true takes effect.  So I am leaving it as-is here.

Reply:
Something wrong here. I'll contibue review and then pull this branch to check.

Reply:
It's weird.  When I comment out IsPackable from Directory.Build.props, it acts as though the ItemGroup DOES realize that IsPackable is true, so the readme file is included for example, but the PropertyGroup does NOT realize it, so Nullable does not get set to enabled.
![image](https://user-images.githubusercontent.com/6377684/157103429-f8806337-c7c1-45f3-8dce-8b90091ead92.png)

Reply:
Please feel free to try to diagnose this.  But if I set IsPackable to true in Directory.Build.props, and IsPackable to false elsewhere as needed, everything works fine.  (With the conditions within Directory.Build.targets)

Reply:
It's also possible the VS IDE is applying IsPackable to the library projects and the compiler is not.  In this case, setting the property manually forces both to operate the same.

Reply:
I pulled branch, removed IsPackable=true,  added <GeneratePackageOnBuild>true</GeneratePackageOnBuild> in NSJ and package was built. Then I added IsPackable=false before GeneratePackageOnBuild and rebuilt project. No nupkg. So all works fine as I can see.

Reply:
I would not change the structure of configuration files moving parts around. We have it the same among few repos. I like that they are the same, because then it is easy to change them at the same time. If the problem is only in one conditional property `nullable` (which you eventually want to apply to all projects) then I suggest move `nullable` under another condition into Directory.Build.props:
```
<PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
    <Nullable>disable</Nullable>
</PropertyGroup>
```

Reply:
> "add true for NSJ"

Sorry, I forgot to escape xml tags; fixed comment.

Reply:
> t's weird. When I comment out IsPackable from Directory.Build.props, it acts as though the ItemGroup DOES realize that IsPackable is true, so the readme file is included for example, but the PropertyGroup does NOT realize it, so Nullable does not get set to enabled.

Because of that ? https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2022:
> Include items in .props files (conditioned on a property). All properties are considered before any item, so user-project property customizations get picked up, and this gives the user's project the opportunity to Remove or Update any item brought in by the import.

Reply:
Nevermind, I'm fine to merge as is and go on.